app/src/main/java/com/remindme/app/navigation/AppNavigation.kt

```diff
@@ -39,6 +39,9 @@ fun AppNavigation() {
         composable(route = Navigation.Routes.TIMELINE) {
             NavigateToTimelineScreen(navController = navController)
         }
+        composable(route = Navigation.Routes.DETAILS) {
+            NavigateToDetailsScreen(navController = navController)
```

Comment:
are we handling back button click also in this PR?

Reply:
it was not there but done now.